docs/_basic/responding_events.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Responding to events
+slug: event-responding
+order: 5
+---
+
+<div class="section-content">
+The easiest way to respond to events is using the `say` function, which exists in any listener middleware function when a channel exists in the event payload. The `say` function accepts a message payload of type string for simple text responses, and of type object for more complex responses.
```

Comment:
this is redundant with the information in `docs/_basic/sending_messages.md`. did you mean to talk about the `respond` argument? if so, then it doesn't make sense in the context of `event()`, but instead should be grouped with `command()` or `action()`.